src/viper/syntax.ml

```diff
@@ -49,7 +49,7 @@ and perm = (perm', info) Source.annotated_phrase
 
 and perm' =
   | WildcardP
-  | FullP
+  | FullP of fldacc
```

Comment:
What is `FullP`? :-)

Reply:
So, none of the named values need to carry any additional parameters, whereas we do need to add an arbitrary permission node that would carry two parameters, e.g., the `p` and `q` in `acc(x.f, p/q)`.